pallets/confidential_transfer/src/lib.rs

```diff
@@ -76,6 +88,11 @@ pub mod pallet {
         }
 
         // SBP-M1 review: missing proper benchmarking
+        /// < HB SBP M2 review
+        ///
+        /// Usually Config items are named more descriptively. In this case i would suggest to rename it to `type Pairing: Paring;`
```

Comment:
There was a copy paste error. What i meant here is that extrinsics lacks weights from benchmarking calculation ( see WeightInfo ) and there is no inline doc. This applies to all the extrinsics reviewed.

Reply:
Thank you.
I am going to work on it as well!